doc/Doxyfile.in

```diff
@@ -31,10 +31,18 @@ MULTILINE_CPP_IS_BRIEF = NO
 INHERIT_DOCS           = YES
 SEPARATE_MEMBER_PAGES  = NO
 TAB_SIZE               = 4
-ALIASES                =
+ALIASES               += addhtml{1}="@htmlonly[block] <div class='newInnerHTML'>\1</div><div> @endhtmlonly"
+ALIASES               += add_toggle{1}="@addhtml{\1} @htmlinclude add_toggle.html \parblock \endparblock"
+ALIASES               += end_toggle="@htmlonly[block] </div> @endhtmlonly"
+ALIASES               += prev_tutorial{1}="**Prev  Tutorial:** \ref \1 \n"
+ALIASES               += next_tutorial{1}="**Next  Tutorial:** \ref \1 \n"
+ALIASES               += add_youtube_link{1}="@htmlonly[block] <div class='addYoutube'>\1</div> @endhtmlonly"
+ALIASES               += youtube{1}="@add_youtube_link{\1} @htmlinclude add_youtube.html \parblock \endparblock"
```

Comment:
Why not replace it with:
```
@htmlonly[block] <iframe width="560" height="315" src="https://www.youtube.com/embed/\1" frameborder="0" allowfullscreen></iframe> @endhtmlonly
```
and use it to add YouTube video by ID: `@youtube 7PF1tAU9se4`?